web/modules/custom/og_addition/tests/scr/Functional/OgAdditionSubscribeLinkTest.php

```diff
@@ -14,94 +14,99 @@ class OgAdditionSubscribeLinkTest extends ExistingSiteBase {
   /**
    * {@inheritdoc}
    */
-  public static $modules = ['node', 'og', 'og_addition'];
+  public static $modules = [
+    'node', 'og', 'og_ui', 'server_general', 'og_addition',
+  ];
 
   /**
    * {@inheritdoc}
    */
-  protected $defaultTheme = 'stark';
+  protected $defaultTheme = 'server_theme';
```

Comment:
Is this required?

Reply:
I was thinking because of the  Drupal\server_general\ElementWrapTrait trait, which uses theme definitions/implementations from the server_theme I needed the theme and the server_general , should I remove og_ui?

Reply:
Please try to follow other tests as much as possible.

Reply:
Sure, I'm finding them resourceful.